deluge/plugins/Stats/deluge/plugins/stats/gtkui.py

```diff
@@ -25,22 +23,30 @@
 from deluge.ui.client import client
 from deluge.ui.gtkui.torrentdetails import Tab
 
-from . import common
+from .common import get_resource
 from .graph import Graph, size_formatter_scale
 
 log = logging.getLogger(__name__)
 
+
 DEFAULT_CONF = {'version': 1,
                 'colors': {
-                    'bandwidth_graph': {'upload_rate': str(gtk.gdk.Color("blue")),
-                                        'download_rate': str(gtk.gdk.Color("green")),
+                    'bandwidth_graph': {'upload_rate':
+                                        str(Gdk.RGBA(red=0.541176, green=0.886275, blue=0.203922, alpha=1.000000)),
```

Comment:
i think i might have tried that at some point in my testing? i can try again but do we have to?